releasenotes/notes/reno-691a6036754487df.yaml

```diff
@@ -0,0 +1,8 @@
+---
+features:
+  - |
+    Added the arguments ``coupling_map`` and ``target`` to :class:`.HighLevelSynthesis`
+    transpiler pass.
+  - |
+    Added the arguments ``coupling_map``, ``target`` and ``qubits`` to :class:`.HighLevelSynthesisPlugin`.
+    This change is backward-compatible.
```

Comment:
Also it might be worthwhile to call out that if they're not explicitly added to a plugin's `run()` method kwargs the coupling map, target, and qubits will end up in the options input.

Reply:
I made a pass over release notes in https://github.com/Qiskit/qiskit-terra/pull/10477/commits/5a1365a7fdb8108f39430e317526d9e1d5307e2e.